src/Servers/Kestrel/Core/src/Internal/LoggerExtensions.cs

```diff
@@ -38,6 +38,6 @@ internal static partial class LoggerExtensions
     [LoggerMessage(7, LogLevel.Error, "The certificate key file at '{CertificateKeyFilePath}' can not be found, contains malformed data or does not contain a PEM encoded key in PKCS8 format.", EventName = "MissingOrInvalidCertificateKeyFile")]
     public static partial void FailedToLoadCertificateKey(this ILogger<KestrelServer> logger, string certificateKeyFilePath);
 
-    [LoggerMessage(8, LogLevel.Warning, "The ASP.NET Core developer certificate is not trusted. For information about trusting the ASP.NET Core developer certificate, see https://aka.ms/aspnet/https-trust-dev-cert.", EventName = "DeveloperCertificateNotTrusted")]
+    [LoggerMessage(8, LogLevel.Warning, "The ASP.NET Core developer certificate is not trusted. For information about trusting the ASP.NET Core developer certificate, see https://aka.ms/aspnet/https-trust-dev-cert", EventName = "DeveloperCertificateNotTrusted")]
```

Comment:
```suggestion
    [LoggerMessage(8, LogLevel.Warning, "The ASP.NET Core developer certificate is not trusted. For information about trusting the ASP.NET Core developer certificate, see https://aka.ms/aspnet/https-trust-dev-cert for more information.", EventName = "DeveloperCertificateNotTrusted")]
```

Reply:
It looks like the other log messages end with periods, so I'd rather rephrase.

Reply:
@amcasey,

> It looks like the other log messages end with periods, so I'd rather rephrase.

I looked around the repo (`LoggerMessage\(.*aka\.ms.*\)`) and the only other two logger messages that mention **aka.ms** links are:

- [`[LoggerMessage(49, LogLevel.Error, "The key ring does not contain a valid default protection key. The data protection system cannot create a new key because auto-generation of keys is disabled. For more information go to http://aka.ms/dataprotectionwarning", EventName = "KeyRingDoesNotContainValidDefaultKey")]`](https://github.com/dotnet/aspnetcore/blob/4d11def519f764caebacdcf726313764eaa1df8e/src/DataProtection/DataProtection/src/LoggingExtensions.cs#L187)
- [`[LoggerMessage(60, LogLevel.Warning, "Storing keys in a directory '{path}' that may not be persisted outside of the container. Protected data will be unavailable when container is destroyed. For more information go to https://aka.ms/aspnet/dataprotectionwarning", EventName = "UsingEphemeralFileSystemLocationInContainer")]`](https://github.com/dotnet/aspnetcore/blob/4d11def519f764caebacdcf726313764eaa1df8e/src/DataProtection/DataProtection/src/LoggingExtensions.cs#L238)

That why I just dropped the **.**.

Your suggestion is closing the sentence with **for more information** when it already starts with **For more information**-